webapp/client/src/stories/InfoForRelativesPage.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import InfoForRelativesPage from "../pages/InfoForRelativesPage";
+import { contentPageDecorator } from "../../.storybook/decorators";
+
+export default {
+  title: "Content Pages/InfoForRelativesPage",
+  component: InfoForRelativesPage,
+  decorators: contentPageDecorator,
+};
+
+function Template() {
+  return <InfoForRelativesPage />;
+}
+
+export const Default = Template.bind({});
```

Comment:
image and text without space:
<img width="1097" alt="Bildschirmfoto 2022-05-02 um 14 09 44" src="https://user-images.githubusercontent.com/37702653/166231413-ab1bad52-f4d1-4a75-abde-8fb8ac899dc5.png">